### DISCUSSION TITLE
First translation into Hungarian

### DISCUSSION THREAD

**Discussion by ghost** [KlipperScreen/KlipperScreen#504] [Ideas]
I would like the first Hungarian translation to be included in the Klipper Screen interface.

I couldn't insert it because I can't create a folder. I will zip the entire file. I would be very grateful if someone could put it in the right folder.

Made by: AntoszHUN

Hungarian Language first translation.
[hu_HU.zip](https://github.com/jordanruthe/KlipperScreen/files/8054659/hu_HU.zip)

**Comment by alfrix** [comment]
it's merged, thank you for the translation

**Reply by ghost** [reply]
> it's merged, thank you for the translation

Thanks for the quick help! I hope it will work. I couldn't test it perfectly. Accented letters are essential. Although, in principle, UTF-8 knows all accented letters. Except for the letters (ő-Ő and ű-Ű).

**Reply by ghost** [reply]
All accented letters appear to be displayed. Very good. It still needs to be repaired. I keep releasing the fixes.

By: AntoszHUN
